src/batch/submit.rst

```diff
@@ -22,11 +26,13 @@ A minimal job script would look something like this:
 
    sort random.txt
 
+|br|
+
 * The first line is a shebang interpreter directive, in this case the interpreter is the *Bash* shell environment.
-* After the shebang any sbatch directives are added. An sbatch directive begins with ``#SBATCH`` followed by the option you want to set. These options can be overriden at execution by supplying them directly to the ``sbatch`` command.
+* After the shebang any sbatch directive is added. An sbatch directive begins with ``#SBATCH`` followed by the option you want to set. These options can be overridden at execution by supplying them directly to the ``sbatch`` command.
```

Comment:
I would change this to "all sbatch directives are added" - there are multiple directives in the example.

Reply:
OK